src/app/state/importaccounts/saga.ts

```diff
@@ -66,6 +66,7 @@ function* enumerateAccountsFromMnemonic(action: PayloadAction<string>) {
 
   try {
     yield* setStep(ImportAccountsStep.LoadingAccounts)
+    // Pre-derive all pages of accounts so we don't need to store mnemonic in redux.
     for (let i = 0; i < accountsPerPage * numberOfAccountPages; i++) {
       const signer = yield* call(oasis.hdkey.HDKey.getAccountSigner, mnemonic, i)
```

Comment:
does the scheme allow us to do the expensive part once instead of 48 times?

Reply:
I didn't think of that :D 

I currently doesn't https://github.com/oasisprotocol/oasis-sdk/blob/8039a75d7fe42dd8021317424cee073e60435f84/client-sdk/ts-web/core/src/hdkey.ts#L33-L35. it's fine

Reply:
https://github.com/oasisprotocol/oasis-sdk/issues/1365 filed